games-board/crafty/crafty-24.1-r2.ebuild

```diff
@@ -1,13 +1,13 @@
 # Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
+
 inherit flag-o-matic toolchain-funcs
 
 DESCRIPTION="Bob Hyatt's strong chess engine"
-HOMEPAGE="http://www.craftychess.com/"
-SRC_URI="http://www.craftychess.com/${P}.zip
-	http://www.cis.uab.edu/hyatt/crafty/source/${P}.zip
+HOMEPAGE="https://www.craftychess.com/"
+SRC_URI=" https://www.craftychess.com/downloads/source/${P}.zip
```

Comment:
The certificate for https://www.craftychess.com expired 250 days ago.

Reply:
Let's use archive.org for now.